### PR TITLE
fix: design module register menubar

### PR DESCRIPTION
**Pull request by bytemain**
### Types

- [x] 🐛 Bug Fixes

### CODE REVIEW

File: packages/design/src/browser/menu-bar/menu-bar.view.tsx [hunk 1]
```diff
@@ -24,16 +24,30 @@ const DesignMenuBarRender = () => {
...
-        x,
-        y: y + height + 4,
+  const handleRefRect = React.useCallback(
+    (cb?: (_anchor: { x: number; y: number }) => void) => {
+      requestAnimationFrame(() => {
+        if (iconRef.current) {
+          const rect = iconRef.current.getBoundingClientRect();
+          const { x, y, width, height } = rect;
+          const _anchor = {
+            x: x + width,
```
Comment (zhanba) [inline packages/design/src/browser/menu-bar/menu-bar.view.tsx:37]: x偏移不需要width

**Review by Ricbet — APPROVED** [review]

**Pull request merged**